content/blog/posts/covid-19-data-scant-during-delta-variant-wave.md

```diff
@@ -0,0 +1,18 @@
+---
+title: COVID-19 data scant during Delta variant wave
+date: 2021-09-20
+authors:
+  - Eva Ruth Moravec
+hero: https://res.cloudinary.com/texas-justice-initiative/image/upload/v1632168497/5_ubwzqr.png
+---
+For months, Texas has been battling the highly contagious Delta strain of the coronavirus. After a surge in the beginning of 2021, [deaths due to COVID-19](https://apps.texastribune.org/features/2020/texas-coronavirus-cases-map/#new-deaths-anchor) were somewhat low in the spring – but then started to climb. 
+
+And although cases have undoubtedly continued to affect populations incarcerated in prisons and jails, where folks are particularly vulnerable, the actual accounting of infections and deaths in Texas lockups has mostly stopped. 
+
+The Texas Commission on Jail Standards, which is responsible for ensuring county jails operate within state law, had for months asked jails to file [daily reports](https://www.tcjs.state.tx.us/wp-content/uploads/2020/12/TA_Memo-COVID_Reporting.pdf) if infections were detected among staff or incarcerated individuals. Yet that practice ended in June, when TCJS informed jails that the reports were no longer required. 
+
+Throughout the pandemic, the Texas Department of Criminal Justice, which oversees prisons, has periodically announced deaths of employees and incarcerated individuals [online](https://www.tdcj.texas.gov/covid-19/index2.html). Yet no deaths of incarcerated individuals have been announced since mid-June. 
```

Comment:
```suggestion
Throughout the pandemic, the Texas Department of Criminal Justice, which oversees prisons, periodically announced deaths of employees and incarcerated individuals [online](https://www.tdcj.texas.gov/covid-19/index2.html). Yet no deaths of incarcerated individuals have been announced since mid-June. 
```